packaging/flavorgen/flavors/generators.go

```diff
@@ -178,6 +173,7 @@ func newCluster(vsphereCluster infrav1.VSphereCluster, controlPlane *controlplan
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      clusterNameVar,
 			Namespace: namespaceVar,
+			Labels:    clusterLabels(),
```

Comment:
Adding this label here so the ClusterResourceSet can use it to find and target this cluster